tests/unit/fixtures/gh870.js

```diff
@@ -1,4 +1,4 @@
-var s;
+var s; // jshint ignore:line
```

Comment:
I think it might be better to not ignore lines and add the errors to the expected.. or change the test to actually use `s`.